src/timeMachine/components/TagSelector.tsx

```diff
@@ -350,6 +409,9 @@ const mstp = (state: AppState, ownProps: OwnProps) => {
     values = [...ADDITIONAL_GROUP_BY_COLUMNS, ...tags.map(tag => tag.key)]
   }
   const isInCheckOverlay = getIsInCheckOverlay(state)
+  const isLast = ownProps.index === tags.length - 1
+  const shouldClearLast =
+    ownProps.index === tags.length - 2 && !tags[tags.length - 1].values.length
```

Comment:
Just wondering, what is the purpose of `shouldClearLast`?

Reply:
there's this weird behavior where if you deselect the last selected list item in a card, and there's another card after the current one, and the next card doesn't have any selected items in it, we should delete that extra card.

Reply:
I believe you mean this, which is a great catch....but we still have it occurring. Since it's an unrelated, found bug -- can we file an issue please?


https://user-images.githubusercontent.com/10232835/189423421-2a1f6ed6-ec47-4d2f-b7fa-cd23be70160a.mov

Reply:
def